tests/service/ProjectAnalysisTests.fs

```diff
@@ -4505,10 +4505,16 @@ module Project35b =
     open System.IO
 
     let fileName1 = Path.ChangeExtension(Path.GetTempFileName(), ".fsx")
+#if DOTNETCORE
+    let fileSource1 = """
+#r "System.Private.CoreLib.dll"
```

Comment:
@KevinRansom For checker.GetProjectOptionsFromScript, is there a duplication of responsibility between `assumeDotNetFramework=false` and `otherFlags = [| "--targetprofile:netcore" |]` ?